lib/sift/value_parser.rb

```diff
@@ -39,6 +40,8 @@ def parse_as_range?
     end
 
     def range_value
+      @value = normalized_date_range if [:date, :time, :datetime].include?(type)
```

Comment:
Do we only want to `normalized_date_range` if the type is set? It looks like it would be safe to try this even if type is not passed in.

Reply:
@amayer171 
That makes sense since range values currently may be only date/time values. 
If the intention is really to support only date/time values as ranges than I'd suggest make more obvious namings for range related variables (like `supports_ranges -> support_datetime_ranges`, `parse_as_range? -> parse_as_datetime_range?` etc)
what do you think?

Reply:
Sounds good to me. We should try to be obvious if the intent of ranges is only for dates. I'm not 100% sure that ranges are only used for dates though.

Reply:
README suggests many different types are supported for ranges: https://github.com/procore/sift#filter-on-ranges

Reply:
looks like it supports only :date, :time, :datetime and :int based on https://github.com/procore/sift/blob/rename-to-sift/lib/sift/type_validator.rb#L25

Reply:
There is conflict within the code base. According to the Parameter class the only types that don't support ranges are `https://github.com/procore/sift/blob/master/lib/brita/parameter.rb#L30` string, text and scope. 

Looks like range validation does not work correctly for those other types of ranges

Reply:
I think we dig to deep out of this pr's scope. 

This PR was introduced because of bug that postgres can't handle datetime like `2018-01-01T10:00:00Z[Etc/UTC]`. 
Intention was to make sift gem a little bit more user-friendly and behave correctly for such values. 
So I suggest in scope of this PR make normalization only for `:datetime` type. 
I just check that it will not work correctly for time 
```
[1] [procore][development] pry(main)> Time.parse("04:05:06")
2019-06-06 04:05:06 +0300
```
And date doesn't have timezone.